src/wp-includes/rest-api/endpoints/class-wp-rest-menu-items-controller.php

```diff
@@ -1018,4 +1034,18 @@ protected function get_menu_id( $menu_item_id ) {
 
 		return $menu_id;
 	}
+
+	/**
+	 * Prime menu item objects.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param WP_Post[] $posts Array of posts
+	 *
+	 * @return WP_Post[]
+	 */
+	public function prime_menu_items_objects( $posts ) {
```

Comment:
The generic name `prime_objects` seems more flexible in case this class gets extended.

Reply:
Can you explain this one a little more? Why is changing the name a benefit here?